### DISCUSSION TITLE
SRT Initial Connection with vMix from iPhone

### DISCUSSION THREAD

**Discussion by hanfordtroy** [HaishinKit/HaishinKit.swift#1640] [Q&A]
We are using your great kit to enable live SRT AV streaming from an iphone to a live vMix session.  It has worked very well thus far. We have one odd observation. When we start publishing, we see packets moving, and we see the SRT connection is established, immediately. Oddly though, we don't end up seeing or hearing video for about 90 seconds on the receive side. Its just black screen in our preview on vMix.  And, then we see the video, but it seems to be in fast-forward-mode for 2-4 seconds, then the video is live, and a perfect solid stream. This could be an issue on the vMix side for sure. I don't think it is bandwidth related, bc the stream plays fine after about a 90 seconds. Anyway, wanted to share my experience. I am happy to help figure this out, if you'd like. Nice Work!

**Comment by shogo4405** [comment]
Is vMix the name of a service? https://www.vmix.com/
Since I'm not familiar with the specifics of the service, it would be best to ask vMix for the reasons behind this issue.

I have tested connections using Ffmpeg, SRT, and Cloudflare but have not encountered such phenomena.
Thank you for your report.

**Reply by hanfordtroy** [reply]
We were able to resolve this issue. I cannot say for sure what our issue was. We looked at your example, modelled your example, with our url info etc confirmed it worked as expected (with vmix), and used it as a model to redevelop a bit of our app. Your examples are excellent, thanks! We should have looked there first.
